### ISSUE TITLE
Miss "extent" in processing input parameters listing

### ISSUE THREAD

**Issue by DelazJ** [qgis/QGIS-Documentation#568]
In [Writing new Processing algo](http://docs.qgis.org/2.8/en/docs/user_manual/processing/scripts.html), the input parameter "extent" is not listed.

It would also be great to have more informations about what are and how to handle *output html* and *output extent* in [handing data produced](http://docs.qgis.org/testing/en/docs/user_manual/processing/scripts.html#handing-data-produced-by-the-algorithm) section, and more globally how to handle the **hidden outputs** (kind of use case?).

**Comment by DelazJ** [comment]
I've created another issue (#668) for the second part of the issue report so this one can be closed once the PR is merged
